_data/scala-compiler-team.yml

```
@@ -0,0 +1,104 @@
+# - name: Core Team
+#   description: Knows everything
+#   image: /resources/img/frontpage/scala-spiral.png
+#   city: Lausanne, Switzerland
+#   email: scala.center@epfl.ch
+#   twitter: scala_lang
+#   github: scala
+#   linkedin: https://www.linkedin.com/in/janedoe/
+#   mastodon: https://fosstodon.org/@janedoe
+#   meetup: https://www.meetup.com/your/url/here
+
+- name: Martin Odersky
+  description: Language Creator
+  github: odersky
+
+- name: Piotr Chabelski
+  description: Product Manager
+  github: Gedochao
+
+- name: Tomasz Godzik
+  description: VirtusLab Compiler Team Leader
```

NIT not sure if necessary, but since we're listing all the roles for everyone, then maybe 🤷‍♂️ 
```suggestion
  description: VirtusLab Compiler Team Leader, Tooling Engineer, Specialised Engineer (IDE)
```